packages/browser/src/login/oidc/IssuerConfigFetcher.spec.ts

```diff
@@ -100,4 +100,24 @@ describe("IssuerConfigFetcher", () => {
     expect(fetchedConfig.scopesSupported).toContain("openid");
     expect(fetchedConfig.scopesSupported).toContain("offline_access");
   });
+
+  it("should return a default value for the supported scopes if not advertized by the OpenID provider", async () => {
+    const fetchResponse = new NodeResponse(
+      JSON.stringify({
+        issuer: "https://example.com",
+        // eslint-disable-next-line camelcase
+        claim_types_supported: "oidc",
+        scopes_supported: undefined,
```

Comment:
The key shouldn't even be in the document to test correctly

Reply:
Good point, fixed in 4915f023f3f87a9e9b0b65c4286aeb2eca30ca68